### PR TITLE
Follow the DataFusion coding style

### PR DESCRIPTION
**Pull request by goldmedal**
# Description
- Used the format style of DataFusion

### CODE REVIEW

**Comment by grieve54706** [comment]
How about adding `cargo fmt`, `cargo install taplo-cli --locked` and `taplo fmt` into `README`?

File: wren-modeling-rs/.gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Cargo.lock
-target/
\ No newline at end of file
+target/
+core/src/main.rs
+sqllogictest/test_sql_files/scratch/
```
Comment (grieve54706) [inline wren-modeling-rs/.gitignore:4]: Please remove unused path

Comment (grieve54706) [inline wren-modeling-rs/.gitignore:4]: Please remove unused path

File: wren-modeling-rs/README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Modeling Module
 
-This is just a very early version of transforming SQL with DataFusion. The main program is a simple use case demonstrating how to use the mdl library.
+This is just a very early version of transforming SQL with DataFusion. The main program is a simple use case
+demonstrating how to use the mdl library.
```
Comment (grieve54706) [inline wren-modeling-rs/README.md:4]: Curious, is the newline a mistake?

File: wren-modeling-rs/README.md [hunk 3]
```diff
@@ -10,7 +11,25 @@ There are some modules:
...
+
+Please format your code with `rustfmt` and `taplo` before submitting a pull request.
+
+## Format with rustfmt
+
+```
+cargo fmt
+```
+
+## Format with taplo
```
Comment (grieve54706) [inline wren-modeling-rs/README.md:30]: IMO, the taplo is only for toml. How about add toml to this line like `Format toml with taplo` to distinguish from rustfmt.

File: wren-modeling-rs/README.md (hunk 2 above)
Comment (goldmedal) [reply wren-modeling-rs/README.md:4]: auto format by IDE. I'll rollback it

**Review by grieve54706 — APPROVED** [review]
LGTM

**Pull request merged**